### DISCUSSION TITLE
请教forestplot画图问题

### DISCUSSION THREAD

**Discussion by zhaoliang0302** [ShixiangWang/self-study#42] [Q&A] [answered]
翔哥好，
我在使用forestplot包画森林图时对于HR范围较大的变量可视化不够理想，虽然这个HR没有意义，我还是想呈现出来。如图
![image](https://user-images.githubusercontent.com/42333702/87451680-ce4c6900-c632-11ea-8855-83d30d002ce8.png)
我想能否截断这个HR范围的显示，这样整个图能舒展一些，但是没搜到forestplot的参数设置，请问这样要怎么操作？
谢谢

**Comment by ShixiangWang** [comment, marked as answer]
我很久没有用这个包了，希望 https://xsliulab.github.io/tumor-immunogenicity-score/#survival-analysis 提到的一些设定可以帮到你

![image](https://user-images.githubusercontent.com/25057508/87561681-9820ef00-c6ef-11ea-8551-6a7ec4f8a0ac.png)

应该是设置 clip

**Comment by zhaoliang0302** [comment]
非常感谢！
